modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2,6 +2,9 @@
 0-analytics-cloud-connection=Analytics Cloud Connection
 1-account-was-added-to-x=1 account was added to {0}.
 1-day=1 Day
+no-objects-created-yet=No Objects created yet
```

Comment:
@rebsilva following Liferay code standards, we must always set language keys as sentences or titles. Sentences must end with a punctuation mark and titles must be capitalized (you can use https://capitalizemytitle.com/). Please, check which one of them this lang key should follow and update it. Thanks :)

Reply:
Thanks Nara! I was following the mockup and didn't realise that!

Reply:
@rebsilva Same comment, could you squash the change here, please?